x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.test.tsx

```diff
@@ -23,49 +39,49 @@ const defaultProps = {
   owner: 'securitySolution',
 };
 
-// FLAKY: https://github.com/elastic/kibana/issues/174525
-// FLAKY: https://github.com/elastic/kibana/issues/174526
-// FLAKY: https://github.com/elastic/kibana/issues/174527
-// FLAKY: https://github.com/elastic/kibana/issues/174528
-describe.skip('CreateCaseFlyout', () => {
-  let mockedContext: AppMockRenderer;
+describe('CreateCaseFlyout', () => {
+  let appMockRenderer: AppMockRenderer;
+
   beforeEach(() => {
-    mockedContext = createAppMockRenderer();
     jest.clearAllMocks();
+    appMockRenderer = createAppMockRenderer();
```

Comment:
Why move them here?

Reply:
Not specific reason, probably a random copy-paste. Do you want me to change the order?

Reply:
No no, I was just wondering if you had found some little performance hack that said moving these to the `beforeEach` block was more efficient 😄 

You can leave as is 👍